exporters/ostream/test/ostream_metric_test.cc

```diff
@@ -147,6 +149,8 @@ TEST(OStreamMetricsExporter, ExportHistogramPointData)
       "\n  type     : HistogramPointData"
       "\n  count     : 3"
       "\n  sum     : 900"
+      "\n  min     : 0"
+      "\n  max     : 0"
```

Comment:
Why are these values 0?

Reply:
It was because `histogram_point_data`, had default min and max.
thanks, added proper min & max now.